data/coalition/coalition missions.txt

```diff
@@ -3211,3 +3211,604 @@ mission "Heliarchs Buy Jump Drive Mission"
+			`	"Can I take a look at your Jump Drive? I'm interested to see how it works," Anna says. You take her to the engine room of <ship>, where she begins taking notes and pictures of the drive.`
+			`	"Well, thank you for bringing me here," Anna says when she finishes. "I didn't sleep too well, so I'll see you later. Goodbye!" She heads back into her cabin, and you settle back into the controls.`
+	npc
+		government "Heliarch"
+		personality heroic opportunistic
+		fleet "Heliarch"
+	on complete
+		set "coalition: struggling author - beyond the hyperdrive"
+		conversation
+			`Anna gives you the book before you both exit the <ship>. The moment you are out, three Kimek approach you and Anna. The Kimek in the middle speaks, and then a brief silence occurs. After a moment, their translation box breaks the silence.`
+			`	"Greetings, <first> <last>. This other human you have brought, who she is, may I ask you?"`
+			`	The Coalition is already familiar with you, so you wait while Anna introduces herself. "About what do you write?" asks the Kimek.`
+			`	"Oh, I write science fiction novels," replies Anna.`
+			`	The Kimek begin conversing amongst themselves in their native language for a good ten to twenty seconds before they move to face you again.`
+			`	Then, switching the translation box back on, the Kimek explains that Anna is welcome to be with you, but reminds you that she will not be allowed to do much unless she registers. "Register?" asks Anna.`
+			`	"You have to register yourself with the Heliarchs before you can do much here. I'll take you to the Ring of Friendship, which is where you can complete the registration process." you say.`
+			`	"Okay," says Anna, "I'll explore the spaceport a little bit to find some inspiration. Get your ship ready and meet me there."`
+
+mission "Coalition: Book Discovered 2"
+	name `"Beyond the Hyperdrive" for Kachachee`
+	description `Now that you have a copy of Anna Fangvin's "Beyond the Hyperdrive," give it to Kachachee and their group on <destination>.`
+	to offer
+		has "coalition: struggling author - beyond the hyperdrive"
+	source
+		government "Coalition"
+		not planet "Ki Patek Ka"
+	destination "Ki Patek Ka"
+	on offer
+		conversation
+			`Sorting through messages, you recall that a Kimek named Kachachee and their group wanted a copy of Anna Fangvin's "Beyond the Hyperdrive." You now know her.`
+			choice
+				`	(Ask Anna for another copy of the book.)`
+				`	(Forget about it.)`
+					decline
+			`	You call Anna, and ask her for another copy of "Beyond the Hyperdrive" for a friend named Kachachee. She says, "Sure, I'll send it to you over the Coalition's incredible hyperspace relays. For something as small as a book, there probably won't even need to be a spaceship bringing it! Where do you want it sent?" You reply, "To <planet>." She hangs up, presumably to ship the second copy of "Beyond the Hyperdrive" to <destination>. You should head there to receive and gift it to Kachachee.`
```

Comment:
Why would you go to Ki Patek Ka to give the book to Kachachee if you think that the book will be sent directly to Kachachee anyways?